src/logic/addressBook/utils/index.ts

```diff
@@ -86,3 +94,19 @@ export const getOwnersWithNameFromAddressBook = (
     }
   })
 }
+
+export const fromAddressListToAddressBookNames = (
```

Comment:
The name of this function it's kinda confusing, shouldn't it be something like "formatAddressListToAddressBookNames"